### PR TITLE
Todo feature

### PR DESCRIPTION
**Pull request by tahiry-dev**
- Create the appearance of the page
- Compile it with Webpack
- Create different components that allow the app to behave correctly
- Use localStorage to store data

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello @tahiry-dev  👋

**You've done pretty well here and a nice UI concept also. 👌**

However, let's make a couple of changes.

- [ ] The localStorage wasn't configured properly in the project. The expected behavior is that after creating a category or todo item, and refreshing the page the todo item and/or category should still remain on the page. Please fix this.

- Please refer to other comments on the code.

### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

File: src/index.js [hunk 1]
```diff
@@ -1,15 +1,73 @@
...
+    case 'add-todo-btn':
+      addNewToDo();
+      renderListContent();
+      break;
+    case 'lists-add-btn':
+      document.body.appendChild(displayNewListBox());
+      break;
+    case 'trash-svg-lists':
+      document.body.appendChild(confirmBoxLists(e.target.parentElement.parentElement));
+      console.log(e.target.parentElement.parentElement);
```
Comment (mikenath223) [inline src/index.js]: - [ ] Kindly remove this console log statement here. Debugging code shouldn't be committed.

File: src/components/toDoController.js [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+        .push({
+          title: getInput('box-title-input'),
+          dueDate: date,
+          done: false,
+          priority: document.querySelector('#box-prio-checkbox').checked,
+        });
+      renderListContent();
+      lightBackground();
+      removeBox();
+      console.log(toDoArray[i].toDos);
```
Comment (mikenath223) [inline src/components/toDoController.js]: - [ ] Also please remove this console.log statement here.

File: src/components/toDoController.js [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+    }
+  }
+
+  date = '';
+}
+/* eslint-enable no-console */
+
+export function setActiveList(targetListId) {
```
Comment (mikenath223) [inline src/components/toDoController.js]: - [ ] Please change this and **all** other regular functions to es6 arrow function syntax.

**Review by arslanbisharat — APPROVED** [review]
# Approved

Thank you for submitting this Pull Request. And thank you for making the requested changes.  Your project is approved. 

Best,
Arslan Bisharat

**Pull request merged**